doc/source/ray-air/benchmarks.rst

```diff
@@ -259,13 +259,13 @@ overhead of a few seconds for both methods that is negligible for longer trainin
 .. _`XGBoost Cluster Configuration`: https://github.com/ray-project/ray/blob/a241e6a0f5a630d6ed5b84cce30c51963834d15b/release/air_tests/air_benchmarks/xgboost_compute_tpl.yaml#L6-L24
 .. _`GPU image batch prediction script`: https://github.com/ray-project/ray/blob/cec82a1ced631525a4d115e4dc0c283fa4275a7f/release/air_tests/air_benchmarks/workloads/gpu_batch_prediction.py#L18-L49
 .. _`GPU image training script`: https://github.com/ray-project/ray/blob/cec82a1ced631525a4d115e4dc0c283fa4275a7f/release/air_tests/air_benchmarks/workloads/pytorch_training_e2e.py#L95-L106
-.. _`GPU prediction small cluster configuration`: https://github.com/ray-project/ray/blob/master/release/air_tests/air_benchmarks/compute_gpu_1_g4_8xl.yaml#L6-L15
-.. _`GPU prediction large cluster configuration`: https://github.com/ray-project/ray/blob/master/release/air_tests/air_benchmarks/compute_gpu_4_g4_12xl.yaml#L6-L15
-.. _`GPU training small cluster configuration`: https://github.com/ray-project/ray/blob/master/release/air_tests/air_benchmarks/compute_gpu_1.yaml#L6-L24
-.. _`GPU training large cluster configuration`: https://github.com/ray-project/ray/blob/master/release/air_tests/air_benchmarks/compute_gpu_16.yaml#L5-L25
+.. _`GPU prediction small cluster configuration`: https://github.com/ray-project/ray/blob/master/release/air_tests/air_benchmarks/compute_gpu_1_cpu_16_aws.yaml#L6-L15
```

Comment:
Can we reference a particular commit?

Reply:
Are your referring something like this?
https://github.com/ray-project/ray/blob/ec2c80dde9104d1fc355c4a0f3a2f0982cdf1056/release/air_tests/air_benchmarks/compute_cpu_1_aws.yaml

Reply:
But in this way, If these file names changed again in the future, it will still pass the test.

Reply:
I guess that's the purpose: If we change the file name it won't break old versions of the doc?

Reply:
Oh ok. 
I'm thinking that if we rename again and forget to update those links in the future, we might forget them forever.

Reply:
Yea, I see your point. Ideally we should point to master in the master branch but to a commit in the release branch. Not sure if there is a way to construct the link programmatically.

Reply:
Can we make a separate PR in the release branch(pin the links to a specific commit)?

Reply:
FWIW it's probably fine to keep the pinned link since this is pointing to the snapshot we ran the benchmarks on.